src/components/progress-bar/progress-bar-styles.jsx

```diff
@@ -0,0 +1,170 @@
+import c from 'color';
+
```

Comment:
Does this feel need to be .jsx?

Reply:
I don't know. I used the button files as templates.